stanford-earth/matson
# Earth 45

# READY FOR REVIEW

# Summary
- Better handler for built-in Drupal messages
- solves the following problems:
  - messages could get covered up by hero banner
  - message styling didn't match the rest of the site
- solves by using the toastr javascript library to overlay messages on top of the page in a graceful, unobtrusive way.  For non-js, creates a new space above the navbar where messages can live
- **@TODO:** added a static message to `docroot/themes/stanford/matson/matson.theme` line 30.  Please remove before merging!

# Needed By current sprint

# Steps to Test

1. Load up any page
2. notice the lovely popup for Drupal system message

# Affected versions or modules
- Does this PR impact any particular module, version, or pull request?  **no**

# Associated Issues and/or People
- JIRA ticket https://stanfordits.atlassian.net/browse/EARTH-45
- Anyone who should be notified? @sherakama 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
